CHANGELOG.md

```diff
@@ -195,6 +195,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Updated CI to only rebuild software dependencies on change
 - Replaced deprecated `optparse.OptionError` with to `click.BadParameter`
 - Relicensed C-PAC from BSD-3-Clause to LGPL-3.0-or-later
+- Updated `FSL-BET` config to default `-mask-boolean` flag as on, and removed all removed `mask-boolean` keys from configs.
```

Comment:
This should go under ["unreleased"](https://github.com/FCP-INDI/C-PAC/blob/fee3e532e773d94811b73dad3964591e6d3d3f56/CHANGELOG.md#unreleased), not under ["1.8.4"](https://github.com/FCP-INDI/C-PAC/blob/fee3e532e773d94811b73dad3964591e6d3d3f56/CHANGELOG.md#changed-3)

Reply:
Ooops sorry, didn't paid much attention there. Thanks for catching this ! Realy appreciate your feedback

Reply:
```suggestion
```

Thanks for adding it to `Unreleased`. We still need to remove it from `1.8.4` though